docs/src/wallets/test-wallets.md

```diff
@@ -27,6 +27,17 @@ You can create a test wallet containing multiple assets (including the base asse
 - coins: `Vec<(UtxoId, Coin)>` has num_assets * coins_per_assets coins (UTXOs)
 - asset_ids: `Vec<AssetId>` contains the num_assets randomly generated `AssetId`s (always includes the base asset)
 
+## Setting up a test wallet with multiple custom assets
+
+You can also create assets with specific `AssetId`s, coin amounts and number of coins.
```

Comment:
```suggestion
You can also create assets with specific `AssetId`s, coin amounts, and number of coins.
```
I'm deeply sorry, it had to be done 😂 

![the-oxford-comma_52c855ed979ed_w1500](https://user-images.githubusercontent.com/3484025/181085624-782cbe70-42e8-4120-8c7b-f572682705a6.jpeg)

Reply:
Will never forget it again :D